nms/1_8_R1Provider/pom.xml

```diff
@@ -16,7 +16,7 @@
         <repository>
             <id>project.local</id>
             <name>local repo</name>
-            <url>file:${project.basedir}/lib</url>
+            <url>file:${project.basedir}\lib</url>
```

Comment:
Why was this changed?

Reply:
That was changed because my IDE wasn't building the project, it said that it can't find the file. Changing that fixed it.

Reply:
This will break the project configuration on any OS aside from Windows :/

Reply:
👌, will revert to the original configuration.

Reply:
Is this even needed? I don't see any dependencies in this module that come from this directory.

Reply:
This is needed for the craftbukkit 1.8 jar, which is needed for building the project. The directory is obviously not committed due to legal reasons, so you have to do it yourself. Check the readme.md file in nms/1_8_R1Provider

Reply:
BuildTools will install the dependency into your local Maven repository, so there's no reason why you need to copy it there.

Reply:
I have it in my local Maven repository, yet IntelliJ says it can't find craftbukkit 1.8.
On another note, it seems that 1.13 bukkit is gone from the Spigot repository for some reason.

Reply:
@DoNotSpamPls You have to make sure you have the right dependency of course. 1.8 had many versions.

Reply:
Yes, I have the exact version required, but it still only finds it when this repository is added and I put the file where it's requested. /shrug

Reply:
Then there's a problem with your setup. The project structure works fine for me. So I'd double check that the jar actually exists, maybe your maven isn't setup properly. But I can assure you the project is.

Reply:
Okay, this is my problem then. Should I remove that section or let it be?

Reply:
I'd say leave it unchanged for now.